### PR TITLE
fix(httplog) update for 3.0 changes

### PR DESCRIPTION
**Pull request by Tieske**
Fixes a badly rendered field (1st commit)

And implements the changes for 3.0, whilst clarifying the description of the older versions (2nd commit)

Original PR with Kong changes: https://github.com/Kong/kong/pull/6992

### CODE REVIEW

File: app/_hub/kong-inc/http-log/_index.md [hunk 1]
```diff
@@ -96,15 +96,10 @@ params:
...
+        An optional table of headers (value indexed by header name) added to the HTTP message to the
+        upstream server. The following headers are not allowed: `Host`, `Content-Length`, `Content-Type`.
 
-        **Note:** This parameter is only available for versions
-        2.3.x and later.
-  extra: |
-    **NOTE:** If the `config.http_endpoint` contains a username and password (for example,
-    `http://bob:password@example.com/logs`), then Kong Gateway automatically includes
-    a basic-auth `Authorization` header in the log requests.
+        **Note:** Before version 3.0.0 the values were arrays of strings (multiple values per header name)
```
Comment (mheap) [inline app/_hub/kong-inc/http-log/_index.md:102]: @Tieske Do you have an example of what the new format should be?

**Review by mheap — COMMENTED** [review]
@lena-larionova Is this a candidate for single sourcing for 3.0?

File: app/_hub/kong-inc/http-log/2.1.x.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+## Custom Headers
+
+The log server that receives these messages might require extra headers, such as for authorization purposes.
+
+```yaml
+...
+  - name: http-log
+    config:
+      headers:
+        Authorization: "Bearer <token>"
```
Comment (Tieske) [inline app/_hub/kong-inc/http-log/2.1.x.md:139]:
```suggestion
        Authorization:
          - "Bearer <token>"
          - "Second Value"
```

File: app/_hub/kong-inc/http-log/_index.md (hunk 1 above)
Comment (Tieske) [reply app/_hub/kong-inc/http-log/_index.md:102]: see the "custom headers" section below. That is the example we currently have, but is broken in the current version. In 3.0 that example will work.

Adding a commit to update the 2.1 version to the proper format with multiple values.

**Comment by lena-larionova** [comment]
Based on the [changelog](https://github.com/Kong/kong/blob/master/CHANGELOG.md) and commit logs for the http-log plugin, combined all versions of the plugins into the index file and tagged each piece of content accordingly.

Since `config.headers` changes in 3.0, I made an old and a new entry for the parameter and tagged the min and max releases. Unfortunately, the plugin config front matter portion doesn't accept jekyll tags, so I can't use conditional rendering inline for a description.

**Review by lena-larionova — APPROVED** [review]

**Pull request merged**